cmd/ethrex_replay/src/block_run_report.rs

```diff
@@ -89,6 +89,36 @@ impl BlockRunReport {
     }
 
     pub fn to_slack_message(&self) -> SlackWebHookRequest {
+        let eth_proofs_button = SlackWebHookActionElement::Button {
+            text: SlackWebHookBlock::PlainText {
+                text: String::from("View on EthProofs"),
+                emoji: false,
+            },
+            url: format!("https://ethproofs.org/blocks/{}", self.number),
+        };
+
+        let mut slack_webhook_actions = vec![SlackWebHookActionElement::Button {
+            text: SlackWebHookBlock::PlainText {
+                text: String::from("View on Etherscan"),
+                emoji: false,
+            },
+            url: if let Network::PublicNetwork(PublicNetwork::Mainnet) = self.network {
+                format!("https://etherscan.io/block/{}", self.number)
+            } else {
+                format!(
+                    "https://{}.etherscan.io/block/{}",
+                    self.network, self.number
+                )
+            },
+        }];
+
+        if let Network::PublicNetwork(_) = self.network {
+            // EthProofs only prove block numbers multiples of 100.
```

Comment:
```suggestion
        if let Network::PublicNetwork(PublicNetwork::Mainnet) = self.network {
            // EthProofs only prove block numbers multiples of 100.
```
I haven't seen blocks from testnets so I think this is correct, is it?